### PR TITLE
image scripts: Support Python 3 and drop Python 2

### PR DESCRIPTION
**Pull request by fabienrohrer**
`hdr.py` was not compliant with Python 3. Supporting also Python 2 is difficult and useless: #1096

### CODE REVIEW

**Comment by fabienrohrer** [comment]
👍

File: scripts/image_tools/images/hdr.py [hunk 1]
```diff
@@ -19,9 +19,11 @@
```
Comment (DavidMansolino) [inline scripts/image_tools/images/hdr.py]: We should probably add the python3 shebang

Comment (fabienrohrer) [reply scripts/image_tools/images/hdr.py]: done

**Comment by fabienrohrer** [comment]
👍

File: scripts/image_tools/images/hdr.py (hunk 1 above)
Comment (omichel) [reply scripts/image_tools/images/hdr.py]: Was it really done? It doesn't seem so...

Comment (fabienrohrer) [reply scripts/image_tools/images/hdr.py]: The Python 3 shebang has been added to the `scripts/image_tools/*py` scripts which are designed to be executed. Adding a shebang to `hdr.py` which is a Python file used by the scripts is meaningless.

So this seems to be correctly solved, yes.

**Pull request merged**